src/js/core.js

```diff
@@ -149,7 +149,7 @@
             return;
         }
 
-        if (MediumEditor.util.isMediumEditorElement(node) && node.children.length === 0) {
+        if (MediumEditor.util.isMediumEditorElement(node) && node.children.length === 0 && !MediumEditor.util.isBlockContainer(node)) {
```

Comment:
I'm using a `<h1>` tag for the editor as I want the user to edit only its content.